controller/src/main/java/org/jboss/as/controller/persistence/yaml/YamlConfigurationExtension.java

```diff
@@ -114,11 +123,15 @@ private void load() {
                         boolean isPresent = config.containsKey(excluded);
                         if (isPresent) {
                             Object value = config.remove(excluded);
-                            String message = MGMT_OP_LOGGER.ignoreYamlElement(excluded);
-                            if (value != null) {
-                                message = message + MGMT_OP_LOGGER.ignoreYamlSubElement(yaml.dump(value).trim());
+                            if (value != null && value instanceof Map && DEPLOYMENT.equals(excluded)) {
```

Comment:
Nit:

value != null && value instanceof Map

can be reduced to

value instanceof Map